docs/contracts/functions.rst

```diff
@@ -79,7 +79,7 @@ Function parameters can be used as any other local variable and they can also be
   parameter. This functionality is possible if you enable the ABI coder v2
   by adding ``pragma abicoder v2;`` to your source file.
```

Comment:
This is quite outdated - abicoder v2 has been the default for a while. We should convert this note into "until version ... it was not possible to...".